packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -21,6 +21,10 @@ const propTypes = {
    * A callback function to execute when the selection state changes. The first parameter is the event. The second parameter is the selectedIndexes.
    */
   onChange: PropTypes.func,
+  /**
+   * A callback function to execute when the selected row loses focus. The first parameter is the event. The second parameter is the index of the row losing focus.
+   */
+  onBlur: PropTypes.func,
```

Comment:
What is the need to introduce this new prop? Could we have done we did for `onClick`?

Reply:
we need blur event to get unselected rows because onChange will be called in onClick method previously selected row loses its focus when new row is selected so calling on-change from wrappedBlur event will gives the row which is newly selected instead of row which lost focus.